src/main/asciidoc/inc/_global-configuration.adoc

```diff
@@ -137,6 +137,9 @@ By default a progress meter is printed out on the console, which is omitted when
 | If set dont push any images even when `{plugin}:push` is called.
 | `docker.skip.push`
 
+| *skipPom*
+| If set to `true` this plugin will skip every projects, where `project.packaging` is set to `pom`. Property: `docker.skip.pom`
```

Comment:
There should be 3 columns in each row:

```
| *skipPom*
| If set to `true` this plugin will skip every projects, where `project.packaging` is set to `pom`.
| `docker.skip.pom`
```

Reply:
@mabrarov: Thanks a lot for noticing, Could you please create a PR to fix this ?

Reply:
Here is it #1448